src/market/config.test.ts

```diff
@@ -0,0 +1,57 @@
+import { DemandConfig } from "./config";
+
+describe("Demand Config", () => {
+  describe("Positive cases", () => {
+    test("It will accept proper config values without an error", () => {
```

Comment:
We are using it() everywhere, not test(). I think keeping only one of those would be better for consistency.